src/index.js

```diff
@@ -473,6 +496,256 @@ const initialize = async () => {
     }
   }
 
+  /**
+   * Ethereum Signature
```

Comment:
Let's just use the RPC method names:
```suggestion
   * eth_sign
```